firmware/application/apps/ui_scanner.cpp

```diff
@@ -791,22 +726,39 @@ void ScannerView::change_mode(freqman_index_t new_mod) {  // Before this, do a s
 
 void ScannerView::start_scan_thread() {
     receiver_model.enable();
-    receiver_model.set_squelch_level(0);
+    receiver_model.set_squelch_level(0);  // ???
```

Comment:
Regarding "???", Scanner checks the signal strength itself in the "on_statistics_update" function against its own squelch settings, in order for squelch to be supported in AM/NFM/WFM (the one in receiver model only works in NFM mode, so I guess the 0 disables it).

Reply:
(Of course it would be great someday if Squelch meant the same thing in every app.)

Reply:
Cool. I added a comment to that effect.